2fa.php

```diff
@@ -86,5 +86,6 @@
 } elseif ($userinfo['type_2fa'] == '2') {
 	// authenticator 2fa enabled
 	$ga_qrcode = $tfa->getQRCodeImageAsDataUri($userinfo['loginname'], $userinfo['data_2fa']);
+        $ga_secret = &$userinfo['data_2fa'];
```

Comment:
why pass that as reference of the variable? Makes absolutely no sense and is not necessary